src/openrct2/config/Config.h

```diff
@@ -245,3 +246,11 @@ void config_set_defaults();
 void config_release();
 bool config_save_default();
 bool config_find_or_browse_install_directory();
+
+bool RCT1DataPresentAtLocation(const utf8* path);
+std::string FindCsg1datAtLocation(const utf8* path);
+bool Csg1datPresentAtLocation(const utf8* path);
+std::string FindCsg1idatAtLocation(const utf8* path);
+bool Csg1idatPresentAtLocation(const utf8* path);
```

Comment:
What's the `i` in `Csg1idatPresentAtLocation` that differs it from `Csg1datPresentAtLocation`? I found the subtle difference very confusing.

Reply:
CSG1i.DAT is the file that contains the indexes for CSG1.DAT. (RCT2 did away with this split, having both the index table and actual sprites in one file.)

Reply:
Ah, ok